### DISCUSSION TITLE
Label 'good first issue' tasks

### DISCUSSION THREAD

**Discussion by srid** [nammayatri/nammayatri#453] [Ideas]
GitHub has an issue label called `good first issue`, which can be applied on issues that are well-suited for newcomers. https://goodfirstissue.dev/

@wololowarrior [asked](https://github.com/nammayatri/nammayatri/pull/398#issuecomment-1499167878):

> how can I start contributing to namma yatri? For starters, I've started learning Haskell. I'm primarily a python/Go dev.
But I'm struggling to identify issues to work on.

**Comment by wololowarrior** [comment]
Thanks! But I am asking specifically for namma yatri. There seems to be only one issue with that tag. 
Maybe the core team can also provide pointers to the files that might involve the change could be at.
Another helpful thing would be an api flow chart. Just to give new joiners a quick high-level intro to various endpoints and their flow e2e.
